app/backend/gwells/views/api.py

```diff
@@ -25,6 +25,7 @@ class GeneralConfig(APIView):
     def get(self, request):
         config = {
             "enable_data_entry": get_env_variable("ENABLE_DATA_ENTRY") == "True",
-            "enable_google_analytics": get_env_variable("ENABLE_GOOGLE_ANALYTICS") == "True"
+            "enable_google_analytics": get_env_variable("ENABLE_GOOGLE_ANALYTICS") == "True",
+            "sso_idp_hint": get_env_variable("SSO_IDP_HINT")
```

Comment:
I think we should go for a default value of "idir" here for now. Since the only case where it's not idir, is for the purposes of functional testing, it's nice to have a default fallback.

e.g.: `get_env_variable("SSO_IDP_HINT", "idir")`

Opinions?

Reply:
definitely agree... IDIR should be the default.

Reply:
IDIR seems like the only sane default.

Reply:
@NickPhura please change to default to "idir".